docs/updating/4-0.md

```diff
@@ -2,6 +2,9 @@
 title: Updating to v4
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```

Comment:
These were missing to begin with. It was evident with the upgrade b/c it would break the site without them.